Pilot/pilotTools.py

```diff
@@ -996,6 +1034,7 @@ def __init__(self):
             ("y:", "CEType=", "CE Type (normally InProcess)"),
             ("z", "pilotLogging", "Activate pilot logging system"),
             ("C:", "configurationServer=", "Configuration servers to use"),
+            ("", "diracXServer=", "DiracX server to use"),
```

Comment:
```suggestion
            ("", "diracx_URL=", "DiracX Server URL to use"),
```
just to be consistent with the DIRACX_URL env variable.